Projects/App/Sources/UI/Organization/OrgDetail/OrgDetailViewController.swift

```diff
@@ -0,0 +1,138 @@
+        orgNameLabel.font = UIFont.systemFont(ofSize: CGFloat(26), weight: .bold)
+        orgNameLabel.textAlignment = .center
+        orgNameLabel.numberOfLines = 2
+        
+        orgInformationSuperStackView.axis = .vertical
+        orgInformationSuperStackView.alignment = .top
+        orgInformationSuperStackView.distribution = . fillProportionally
+        orgInformationSuperStackView.spacing = 50
+        
+        orgInformationStackView1.axis = .horizontal
+        orgInformationStackView1.alignment = .center
+        orgInformationStackView1.distribution = .fillEqually
+        orgInformationStackView1.spacing = 30
+        
+        orgInformationStackView2.axis = .horizontal
+        orgInformationStackView2.alignment = .center
+        orgInformationStackView2.distribution = .fillEqually
+        orgInformationStackView2.spacing = 30
+
+        orgInformationStackView3.axis = .horizontal
+        orgInformationStackView3.alignment = .center
+        orgInformationStackView3.distribution = .fillEqually
+        orgInformationStackView3.spacing = 30
+
+        orgInformationImage1.image = UIImage(.img_reviewfriends_together)
+        orgInformationImage1.contentMode = .scaleAspectFit
+        orgDetailInformationView1.informationLabel.text = "함께하는 친구들"
+        orgDetailInformationView1.numberLabel.text = "13,966명"
+        
+        orgInformationImage1.image = UIImage(.img_categoryfriends_western)
+        orgInformationImage1.contentMode = .scaleAspectFit
+        orgDetailInformationView2.informationLabel.text = "등록된 맛집"
+        orgDetailInformationView2.numberLabel.text = "1,425곳"
+        
+        orgInformationImage3.image = UIImage(.img_reviewfriends_photo)
+        orgInformationImage3.contentMode = .scaleAspectFit
+        orgInformationImage3.layer.applyFigmaShadow(color: .black, opacity: 0.05, xCoord: 0, yCoord: 0, blur: 5, spread: 0)
+        orgDetailInformationView3.informationLabel.text = "업로드된 사진"
+        orgDetailInformationView3.numberLabel.text = "124,513개"
+        
+        orgInviteButton.setTitle("우리학교 사람들 초대하기", for: .normal)
+    }
+
+    private func createLayout() {
+        view.addSubviews([orgNameLabel, orgInformationSuperStackView, orgInformationStackView1, orgInformationStackView2, orgInformationStackView3, orgInviteButton])
+        orgInformationSuperStackView.addArrangedSubviews([orgInformationStackView1, orgInformationStackView2, orgInformationStackView3])
+        orgInformationStackView1.addArrangedSubviews([orgInformationImage1, orgDetailInformationView1])
+        orgInformationStackView2.addArrangedSubviews([orgInformationImage1, orgDetailInformationView2])
+        orgInformationStackView3.addArrangedSubviews([orgInformationImage3, orgDetailInformationView3])
+
+        orgNameLabel.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide).inset(30)
+            make.centerX.equalToSuperview()
+            make.height.equalTo(31)
```

Comment:
p2;

라벨은 intrinsic content size가 있어서 높이 지정 안해줘도 될걸요?

<img width="250" src="https://user-images.githubusercontent.com/75792767/198208664-8c94bf4a-bdef-4b4d-8a92-6ad8f3a95219.png"><img width="250" src="https://user-images.githubusercontent.com/75792767/198208704-4f99ee43-6a37-4eb6-a549-c275216dfb03.png">

그리고 지금 좌우 제약조건이 없어서 캠퍼스명이 길어지면 화면을 뚫게 되니까 좌우 제약조건 만들어주세요~


```suggestion
            make.horizontalEdges.equalToSuperview().inset(40)
```

Reply:
[d1f354a](https://github.com/DeveloperAcademy-POSTECH/MacC-TEAM-ZESTY/pull/70/commits/d1f354ad3efc3349b294ec018a2c4a204eacdd70) 섬세함 땡큐 ~